### DISCUSSION TITLE
样式化是否有更加详细的示例

### DISCUSSION THREAD

**Discussion by jdysya** [vuepress-theme-hope/vuepress-theme-hope#3070] [Q&A] [answered]
对于这个[如何进行文本块的链接](https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/3069)，我目前的想法是通过[样式化](https://vuepress-theme-hope.github.io/v2/guide/markdown/stylize.html)，我尝试写了一个正则表达式，但是并没有效果，尽管打开`debug`模式，也无法在控制台打印相关的内容

![image](https://user-images.githubusercontent.com/73117213/233974097-59a1bf29-01b6-4073-bf8c-7bc73cbae9f7.png)
> 这个样例只是一个测试，我只是想匹配类似于`^test`这样的内容，然后将其转换为`<a href="#test"></a>`这样的超链接，然而这个正则表达式并没法匹配到`^test`

**Comment by Mister-Hope** [comment, marked as answer]
https://regex101.com/ 先测测你写的正则是匹配什么的吧

**Reply by jdysya** [reply]
这个网站显示匹配没问题
<img width="893" alt="image" src="https://user-images.githubusercontent.com/73117213/233984686-cf2a4b03-1868-47a7-be92-1265f282f4be.png">
<img width="534" alt="image" src="https://user-images.githubusercontent.com/73117213/233984733-f0d49a0f-0890-4f4a-b993-44e887c8a21a.png">
结果并不符合预期
<img width="1101" alt="image" src="https://user-images.githubusercontent.com/73117213/233984806-6446f323-844e-4872-bee9-38c46a4b515b.png">
